### PR TITLE
v3.4.1 is available and has no known security issues https://snyk.io/…

### PR DESCRIPTION
**Pull request by d3k4z**
…test/npm/bootstrap/3.4.1

### CODE REVIEW

**Comment by samdark** [comment]
Why fixing the version? 3.4.* should install 3.4.1 anyway, shouldn't it?

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     ],
     "require": {
         "yiisoft/yii2": "~2.0.6",
-        "bower-asset/bootstrap": "3.4.* | 3.3.* | 3.2.* | 3.1.*"
+        "bower-asset/bootstrap": "3.4.1 | 3.3.* | 3.2.* | 3.1.*"
```
Comment (bizley) [inline composer.json:36]:
```suggestion
        "bower-asset/bootstrap": "~3.4.1 | 3.3.* | 3.2.* | 3.1.*"
```

**Comment by bizley** [comment]
3.4.0 contains vulnerability, we should disallow it.

**Comment by rob006** [comment]
> 3.4.0 contains vulnerability, we should disallow it.

No, we should not. This is not a framework job to work as a guard for such cases. At worst case `~3.4.1` will just prevent `yiisoft/yii2` update, because Composer is smart enough to use lower version if there is a conflict.

**Comment by samdark** [comment]
I agree with @rob006.

**Comment by samdark** [comment]
Thanks for contributing, @d3k4z but won't merge it.

**Pull request closed without merging**